plasmapy/formulary/tests/test_ionization.py

```diff
@@ -3,17 +3,35 @@
 import pytest
 from astropy.constants import c
 
-from plasmapy.formulary.ionization import Z_bal
+from plasmapy.formulary.ionization import Z_bal_, Saha
 
 
-def test_Z_bal():
+def test_Z_bal_():
     n = 1e19 * u.m ** -3
     T_e = 5000 * u.K
 
-    assert (Z_bal(n, T_e) * u.dimensionless_unscaled).unit == u.dimensionless_unscaled
+    assert (Z_bal_(n, T_e) * u.dimensionless_unscaled).unit == u.dimensionless_unscaled
 
     with pytest.warns(u.UnitsWarning):
-        Z_bal(1e19, T_e)
+        Z_bal_(1e19, T_e)
 
     with pytest.raises(u.UnitTypeError):
-        Z_bal(2e19 * u.kg, T_e)
+        Z_bal_(2e19 * u.kg, T_e)
+
+
+def test_Saha():
```

Comment:
Should add at least one value check in this test.  It could be a repeat of doctest.

I see the tests defined in `tests_*.py` as the complete test suite for a given functionality, and doctests are in addition to them.  A complete test suite should not be split among multiple locations.

Reply:
how do I add a value check to the tests? I've looked at other test functions but can't find any that do that.

Reply:
This isn't the cleanest example four our case here, but [this is how I did it for `validate_quantites`](https://github.com/PlasmaPy/PlasmaPy/blob/511c25bce06fda6e393e03fc1d422901e8779591/plasmapy/utils/decorators/tests/test_validators.py#L479-L556).  `validate_quantities` is basically an alias for `ValidateQuantities`.  The idea is, you use the `mock` package to mock `ionization_state` and that will allow you to check if `ionization_state` was called when `Z_bal_` was used and what arguments were passed.

Reply:
So, ignore everything I said about mocking.  Since our alias is not a function calling a function, i.e.

```python
def Z_bal(*args, **kwargs):
    return ionization_balance(*args, **kwargs)
```
we do not have to do any fancy mocking.  You can test the alias just by doing...
```python
assert Z_bal_ is ionization_balance
```

Reply:
I'm so sorry, I just realized we're talking about two different things.  Completely my bad.  See my [comment below](https://github.com/PlasmaPy/PlasmaPy/pull/860/files#r452641106) for both the value and unit tests.